x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plan/physical/PhysicalPlan.java

```diff
@@ -13,12 +13,6 @@
 
 import java.util.List;
 
-/**
```

Comment:
should this stay?
also, does this make sense for QL?

Reply:
No, that's an accidental commit that needs removal.